src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -827,15 +827,25 @@ public Payments getPaymentsForAccount(final UUID accountId, final AuditLevel aud
         return httpClient.doGet(uri, queryParams, Payments.class);
     }
 
-    public Payment createPayment(final ComboPaymentTransaction comboPaymentTransaction, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment) throws KillBillClientException {
+    public Payment createPayment(final ComboPaymentTransaction comboPaymentTransaction, List<String> controlPluginNames, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment) throws KillBillClientException {
         final String uri = JaxrsResource.PAYMENTS_PATH + "/" + JaxrsResource.COMBO;
-        final Multimap<String, String> params = HashMultimap.<String, String>create();
-        final Multimap<String, String> queryParams = paramsWithAudit(params,
-                                                                     createdBy,
-                                                                     reason,
-                                                                     comment);
-        storePluginPropertiesAsParams(pluginProperties, params);
-        return httpClient.doPostAndFollowLocation(uri, comboPaymentTransaction, queryParams, Payment.class);
+
+        final Multimap<String, String> queryParams = HashMultimap.create();
```

Comment:
Can you keep `HashMultimap.<String, String>create()`? Type inference is broken on old 1.6 jdk versions.